### ISSUE TITLE
Clarification on the /versions C-S Api endpoint and how the versioning works

### ISSUE THREAD

**Issue by MTRNord** [matrix-org/matrix-spec#1698]
**Link to problem area**: https://spec.matrix.org/v1.8/client-server-api/#get_matrixclientversions and https://spec.matrix.org/v1.8/#specification-versions

**Issue**
There seem to be at least 2 if not 3 different interpretations on this. Js-Sdk thinks that implementing 1.8 also requires implementation of 1.1. But from a new implementor perspective, it feels odd to implement all versions from 1.1 until 1.8. It not only slows down the implementation but also makes it at least 4x as hard.

This is a little different than room versions which exist in parallel. Also I am aware that obviously there will be an overlap on the update periods since sdks have to catch up. However it feels odd to needing to implement almost over 2 years of specifications for a server to work.

The relevant js-sdk issue is at https://github.com/matrix-org/matrix-js-sdk/issues/3915 (There is some more info but to avoid unnecessary duplication I won't copy that over here)

**Comment by richvdh** [comment]
> There seem to be at least 2 if not 3 different interpretations on this. Js-Sdk thinks that implementing 1.8 also requires implementation of 1.1. But from a new implementor perspective, it feels odd to implement all versions from 1.1 until 1.8

This is no longer correct. As of the current writing (and https://github.com/matrix-org/matrix-js-sdk/pull/4014), matrix-js-sdk will accept a server which claims to implement *any* spec version in the range 1.1 to 1.8.

The spec could be clearer about how this is meant to work, so feel free to open a new issue with refrerence to https://github.com/matrix-org/matrix-js-sdk/issues/3915#issuecomment-1865221366, but the premise of this issue seems to be incorrect, so I'm going to close it.
